### ISSUE TITLE
Inline Lity (pop-up) and Youtube Video with Specific Start Time

### ISSUE THREAD

**Issue by gerard-pcom** [jsor/lity#232]
TLDR: **How do you make youtube video URLs with specific start time to work on pop-up/inline Lity?**

Portion of my code:
`<div id="inline" style="background:#fff" class="lity-hide"></div>`
`<a href="<?php esc_url($video_url); ?>" data-lity="">IMG</a>`

Sample $video_url values that worked:
`https://youtu.be/A_WzVVndUCY`
`https://www.youtube.com/embed/A_WzVVndUCY`
`https://www.youtube.com/watch?v=A_WzVVndUCY?autoplay=0&rel=0&showinfo=0&autohide=1&modestbranding=1`

Sample $video_url values that did NOT work:
`https://youtu.be/A_WzVVndUCY?t=181`
`https://youtu.be/A_WzVVndUCY?start=181`
`https://www.youtube.com/embed/A_WzVVndUCY?start=181`
`https://www.youtube.com/embed/A_WzVVndUCY?start=181;rel=0&amp;showinfo=0`
`https://www.youtube.com/embed/A_WzVVndUCY?start=181&end=201;rel=0&amp;showinfo=0`
`https://youtu.be/A_WzVVndUCY?t=181&autoplay=0&rel=0&showinfo=0&autohide=1&modestbranding=1`

The ones that did not work only worked when I entered them in the browser address bar. Just to make sure, I did a var_dump to check the value of the variable ($video_url).

This is the image of the video with a var_dump expression before it:
![Screen Shot 2019-12-16 at 2 54 51 PM](https://user-images.githubusercontent.com/20609119/70885472-e19f4380-2013-11ea-9d34-cb7139f45f9d.png)

This is the pop-up/inline Lity but it doesn't start at the specified start time:
![Screen Shot 2019-12-16 at 2 52 22 PM](https://user-images.githubusercontent.com/20609119/70885322-85d4ba80-2013-11ea-80e6-7e9c2b8a0768.png)

**Comment by kevinohashi** [comment]
Was this ever fixed? Running into same issue now.
